src/apps/service/apps/de/jobs.clj

```diff
@@ -95,17 +91,26 @@
     :username        (:username user)
     :wiki_url        (:wiki_url jex-submission)}))
 
+(defn- submit-job-in-batch
+  [user submission job]
+  (->> (try+
+        (do-jex-submission job)
+        (save-job-submission user job submission)
+        (catch Object _
+          (save-job-submission user job submission "Failed")))
+       (format-job-submission-response user job true)))
+(defn- submit-standalone-job
+
```

Comment:
Heh, I think I transposed these two lines. I really need to disable that Emacs keybinding. 😆